### DISCUSSION TITLE
[Embedded] Keep the zoom level of a graph to use again in editing

### DISCUSSION THREAD

**Discussion by jdelpierre** [jgraph/drawio#2628] [General]
* [x] I agree to follow the [Code of Conduct](https://github.com/jgraph/drawio/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for a feature request that matches the one I want to file, without success.

Hello, 

I would like to be able to open the editor with a particular zoom (knowing that it is an embedded environment.
When the user chooses a 50% zoom, I would like to know if it is possible to recover this value to store it, and use it again when he comes back on this editor, or even by graph directly?
Because each time, we end up at 100%.

Thanks for your answers

**Comment by davidjgraph** [comment]
If we stored the zoom in the file, each time a different user opened it they would get the zoom of the last user.

**Reply by jdelpierre** [reply]
Thanks for this information,

If this was stored in the localstorage configurations? Would it be unique per user? 
And when we switch to embedded version, it is not possible to retrieve this information by a postMessage() event?
Which would then also allow to send this value back to the editor whenever you want? 

Because some canvas that are visible correctly on 50% zoom for example, at each opening it is not comfortable to return to 100%.

Thanks
